CONTRIBUTING.md

```diff
@@ -22,20 +22,43 @@ To make a change to the FVM.
 
 ## Releasing
 
-To prepare a release.
-
-1. Make sure the crate's `CHANGELOG.md` is up-to-date (look through `git log -- path/to/crate`).
-2. Make sure the changes are well tested. See [builtin actors testing][builtin actors testing].
-3. Pick a next version (for each crate you're releasing). Look through the changelogs and carefully
-   decide if it's a breaking release or not. Then read [rust's semver
-   documentation](https://doc.rust-lang.org/cargo/reference/semver.html).
-4. Install `cargo-edit` (`cargo install cargo-edit`).
-5. Create a new branch.
-6. Use `cargo set-version` to set the version for each crate you're releasing. This will both update
-   the crate version, and make all other crates in the workspace depend on the latest version.
-7. Make sure the changelogs are all up-to-date, set the release date & version, and add a new
-   "Unreleased" section.
-8. Commit your changes, and make a PR.
+The primary crates, `fvm`, `fvm_shared`, `fvm_sdk`, and `fvm_integration_tests`, are versioned and
+released together. Other crates in the workspace are versioned and released independently.
```

Comment:
I'd extend this a bit:

- The primary crates (`fvm`, `fvm_shared`, and `fvm_sdk`) and the integration testing framework (`fvm_integration_tests`) are released together.
- The `fvm_ipld_*` crates (living in `ipld/`) are released independently and only live in this repo for convenience.
- The rest of the crates are for local testing and are not released.